### DISCUSSION TITLE
Add a JSON rendering plugin

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5833] [Feature Requests]
>Does S7 have a JSON rendering component for the UI? This could be useful to render pretty json data put into a text field.

**Requested By:** Matthew Cruz at The University of Michigan

**Comment by maxpatiiuk** [comment]
No, no such thing at the moment.
Could you ask him for use cases?
Would folding/unfolding be needed too?
Would proper color highlighting be needed?
If so, instead of rendering this as a text box, we could just borrow the editor from the app resources
